core/src/main/java/org/apache/carbondata/core/datamap/DataMapFilter.java

```diff
@@ -131,6 +149,6 @@ public boolean isResolvedOnSegment(SegmentProperties segmentProperties) {
         .hasColumnDriftOnSegment(table, segmentProperties)) {
       return false;
     }
-    return true;
+    return true;*/
```

Comment:
please revert to old code

Reply:
done. It was added to run CI. removed now